modules/core/02-client/keeper/events.go

```diff
@@ -101,3 +103,14 @@ func EmitSubmitMisbehaviourEvent(ctx sdk.Context, clientID string, clientState e
 		),
 	)
 }
+
+// EmitUpgradeChainEvent emits an upgrade chain event.
+func EmitUpgradeChainEvent(ctx sdk.Context, height int64) {
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeUpgradeChain,
+			sdk.NewAttribute(types.AttributeKeyUpgradePlanHeight, strconv.Itoa(int(height))),
```

Comment:
In this [PR](https://github.com/cosmos/ibc-go/pull/1160/files) Fede recommended to use `strconv.FormatUint` for converting `uint64` to `string`. Could we not use `strconv.FormatInt` instead here? It's not like I have a strong opinion for using one or the other way, but I guess it would be nice to settle in one (if possible) and use that always. :)

Reply:
The type is `int64` but there is also a `strconv.FormatInt(height, 10)` we can use instead!